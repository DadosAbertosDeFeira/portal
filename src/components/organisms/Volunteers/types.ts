import type { Volunteer } from "@utils/volunteers";
import type { BoxProps } from "atoms/Box";

export type VolunteersProps = BoxProps & {
  volunteers: Volunteer[];
};
