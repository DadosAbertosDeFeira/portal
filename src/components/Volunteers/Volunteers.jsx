import React from 'react';
import Image from 'next/image';

import VolunteersCard from '../VolunteersCard';
import volunteersList from '../../utils/volunteers';
import HandWithHeart from '../../../public/assets/icons/hand-with-hearth.svg';

const Volunteers = () => {
  return (
    <>
      <div className="flex flex-col items-center">
        <Image
          className="max-w-[100px] max-h-[100px]"
          src={HandWithHeart}
          alt="Livro"
        />
        <div className="m-lg text-center">
          <h2 className="tablet:m-xl">Volunt√°rios</h2>
          <h3 className="tablet:m-xl font-semi">Quem fez acontecer</h3>
        </div>
      </div>
      <div className="flex flex-wrap justify-center mt-12 gap-4">
        {volunteersList.map((volunteer) => (
          <VolunteersCard
            key={`${volunteer.name}-${volunteer.role}`}
            name={volunteer.name}
            role={volunteer.role}
            picture={volunteer.picture}
            linkedin={volunteer.linkedin}
            instagram={volunteer.instagram}
            github={volunteer.github}
            twitter={volunteer.twitter}
          />
        ))}
      </div>
    </>
  );
};

export default Volunteers;
