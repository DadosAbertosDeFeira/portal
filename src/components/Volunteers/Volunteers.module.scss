.image {
  width: 100px;
  height: 100px;
}

.volunteersList {
  display: flex;
  justify-content: center;
  flex-wrap: wrap;

  margin-top: 3em;
  gap: 1em;
}
