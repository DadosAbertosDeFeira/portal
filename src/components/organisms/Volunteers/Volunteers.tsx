import Hand from "@assets/icons/hand-with-hearth.svg";
import type { SocialMediaType } from "@utils/social";
import { Image } from "atoms/Image";
import { SocialMediaCard } from "molecules/SocialMediaCard";
import { TitleSection } from "molecules/TitleSection";
import React from "react";

import type { VolunteersProps } from "./types";

export function Volunteers({ volunteers }: VolunteersProps) {
  return (
    <TitleSection
      src={Hand}
      className="p-2 md:px-10"
      alt="Mão com um coração"
      title="Voluntários"
    >
      <ul className="grid auto-rows-[115px] grid-cols-[repeat(auto-fill,_minmax(260px,_1fr))] items-center justify-items-center gap-8">
        {volunteers.map(({ name, picture, role, ...medias }) => (
          <li
            className="flex h-full w-full rounded-[4px] border-t-[3px] border-solid border-blue-500 p-4 shadow-lg"
            key={name}
          >
            <Image
              src={picture}
              alt={name}
              width={70}
              height={70}
              className="mr-4 h-[70px] w-[70px] self-center rounded-[50%] bg-white"
            />
            <div className="flex flex-col justify-between gap-2 overflow-hidden">
              <h3 className="font-normal leading-tight text-black">{name}</h3>
              <p className="text-md truncate leading-tight text-blue-500">
                {role}
              </p>
              <ul className="flex flex-row items-center gap-x-1">
                {Object.entries(medias).map(([key, value]) => {
                  return (
                    <SocialMediaCard
                      key={key}
                      type={key as SocialMediaType}
                      user={value}
                    />
                  );
                })}
              </ul>
            </div>
          </li>
        ))}
      </ul>
    </TitleSection>
  );
}
