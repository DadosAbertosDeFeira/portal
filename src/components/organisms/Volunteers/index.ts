export * from "./types";
export * from "./Volunteers";
