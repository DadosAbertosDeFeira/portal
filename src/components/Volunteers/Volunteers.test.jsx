import React from 'react';
import { render } from '@testing-library/react';

import Volunteers from './Volunteers';

describe('<Volunteers />', () => {
  it('renders correctly the component', () => {
    const { asFragment, getByText } = render(<Volunteers />);

    expect(getByText('Volunt√°rios')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
