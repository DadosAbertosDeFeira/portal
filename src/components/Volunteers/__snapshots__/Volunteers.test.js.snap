// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Volunteers /> renders correctly the component 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col items-center"
  >
    <img
      alt="Livro"
      class="image"
      src="/assets/icons/hand-with-hearth.svg"
    />
    <h2
      class="m-lg tablet:m-xl"
    >
      Voluntários
    </h2>
  </div>
  <div
    class="volunteersList"
  >
    <div
      class="volunteers"
    >
      <img
        alt="Allythy"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/allythy"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Allythy
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://github.com/allythy"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Ana Paula Gomes"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/anapaulagomes"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Ana Paula Gomes
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Faz-tudo
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/anapaulagomess"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/anapaulagomess"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/anapaulagomes"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="André Costa"
        class="volunteersPicture"
        src="https://media.licdn.com/dms/image/C4D03AQFDIUn_R4yx2Q/profile-displayphoto-shrink_400_400/0/1668127163554?e=1681344000&v=beta&t=SM5dXuDYUW-HKvk37wmbRuXbTmSKFVJ6Yo3q_-9pkhs"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            André Costa
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Designer
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/andrecostaux"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Bruno Morassuti"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/jedibruno"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Bruno Morassuti
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Conselheiro Jurídico
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/555112299jedi"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/bruno-schimitt-morassutti-7a0b5b180"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/jedibruno"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Daniel Fireman"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/danielfireman"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Daniel Fireman
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Conselheiro
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/daniellfireman"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/danielfireman"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/danielfireman"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Eduardo Cuducos"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/cuducos"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Eduardo Cuducos
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/cuducos"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://github.com/cuducos"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Eduardo Daltro"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/u/12780445?v=4"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Eduardo Daltro
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Staff
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/daltroedu"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Emerson Oliveira"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/emersonsdo"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Emerson Oliveira
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/emerson-oliveira-89ab859"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/emersonsdo"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Everton Alves"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/ellalves"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Everton Alves
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Social Media
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.instagram.com/ellalves.mktd"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Instagram icon"
              src="/assets/icons/instagram.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Felipe Barros"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/felipelfb"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Felipe Barros
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Analista de Dados
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/felipe-barros-609a52141"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/felipelfb"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Filipe Maia"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/filipebsmaia"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Filipe Maia
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/filipebsmaia"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/filipebsmaia"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Gabriela Turquetti"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/turquetti"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Gabriela Turquetti
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedora
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/gabturquetti"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/gabriela-turquetti"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://www.instagram.com/gabrielaturquetti"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Instagram icon"
              src="/assets/icons/instagram.svg"
            />
          </a>
          <a
            href="https://github.com/turquetti"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Geraldo Castro"
        class="volunteersPicture"
        src="https://pbs.twimg.com/profile_images/1402724348507701252/3-SBqUzV_400x400.jpg"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Geraldo Castro
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/exageraldo_"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/exageraldo"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/exageraldo"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Guilherme Wanderley"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/guilhermelowa"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Guilherme Wanderley
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Office-boy e Analista de Dados
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/guilherme-wanderley"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/guilhermelowa"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Luanda Dantas"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/luandadantas"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Luanda Dantas
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedora
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/luandadantas"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/luandadantas"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Matheus Rocha"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/matheusrocha89"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Matheus Rocha
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/matheusrocha"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/matheus-cruz-rocha-38021525"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/matheusrocha89"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Nataly Costa"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/natalycosta"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Nataly Costa
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            UI/UX design manager
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/natalycosta"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Nathanael Souza"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/u/91571451?s=400&u=df8fba5c5ee9f61fca7027db2d22011429152be2&v=4"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Nathanael Souza
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/nathanael-souza"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/Nathanaelss"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Rafael Gomes"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/gomex"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Rafael Gomes
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Desenvolvedor
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/gomex"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/rbgomes"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/gomex"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Rafael Velame"
        class="volunteersPicture"
        src="https://media.licdn.com/dms/image/C4D03AQG8ytEVvsZ44g/profile-displayphoto-shrink_100_100/0/1519088249816?e=1681344000&v=beta&t=mzqZkJbugdXi6Su20QsyLj8O0FscPrhFvS-lOknUjYg"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Rafael Velame
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Jornalista
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.twitter.com/rafaelvelame"
            target="_self"
          >
            <img
              alt="Twitter icon"
              src="/assets/icons/twitter.svg"
            />
          </a>
          <a
            href="https://www.linkedin.com/in/rafael-velame-242b98104"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
        </div>
      </div>
    </div>
    <div
      class="volunteers"
    >
      <img
        alt="Vivian Macedo"
        class="volunteersPicture"
        src="https://avatars.githubusercontent.com/u/7208924?v=4"
      />
      <div
        class="volunteersInfo"
      >
        <div>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoName"
            data-testid=""
          >
            Vivian Macedo
          </p>
          <p
            class="text-black text-sm tablet:text-base volunteersInfoRole"
            data-testid=""
          >
            Analista de Dados
          </p>
        </div>
        <div
          class="volunteersIconContainer"
        >
          <a
            href="https://www.linkedin.com/in/vivianmacedo"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Linkedin icon"
              src="/assets/icons/linkedin.svg"
            />
          </a>
          <a
            href="https://github.com/vihvs"
            rel="noreferrer"
            target="_blank"
          >
            <img
              alt="Github icon"
              src="/assets/icons/github.svg"
            />
          </a>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
